core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -676,4 +677,24 @@ default EventExecutor defaultSubscriberExecutor() {
         requireNonNull(function, "function");
         return new RecoverableStreamMessage<>(this, function, /* allowResuming */ true);
     }
+
+    /**
+     *  Provide general way to use {@link StreamMessages} by applying the specified {@link Function}.
```

Comment:
/** 
  * Writes this {@link StreamMessage} to the given {@link Path} with {@link OpenOption}s.
  * See {@link StreamMessages#writeTo} for the details.